llvm/include/llvm/Support/GenericLoopInfoImpl.h

```diff
@@ -645,6 +645,36 @@ LoopInfoBase<BlockT, LoopT>::getLoopsInReverseSiblingPreorder() const {
   return PreOrderLoops;
 }
 
+template <class BlockT, class LoopT>
+LoopT *LoopInfoBase<BlockT, LoopT>::getSmallestCommonLoop(LoopT *A,
```

Comment:
[not a change request] I know the function name comes from `Cycle`, but usually "smaller" and "larger" does not apply to loops. Does it refer to number iterations? number of instructions in body? Execution time?